### DISCUSSION TITLE
Use days different color to mark weekend dates.

### DISCUSSION THREAD

**Discussion by lukaszlew** [iSoron/uhabits#851] [Feature Requests]
Thank you for the app. Polish level is great!
The following is feature request / idea:

While I realize that for some people Sat and Sun are no different than Mon-Fri, I believe that for most, the routines (and desired routines) are rather different. I think it would make sense to have a slightly different shade of background for weekend columns or at least headers. What do you think?

**Comment by hiqua** [comment]
It would be nice, with the caveat that in some countries the beginning of the week is considered to be Sunday and having a different background on weekends could be misleading because of this. I think it would be nice to have just to scroll back in the past more easily.

**Reply by lukaszlew** [reply]
Is it sufficient to rely on the option in Settings selecting first day of the week?

**Reply by hiqua** [reply]
I forgot about this option, yes indeed, good point!

**Reply by herrbrixter** [reply]
First day of the week and weekends should be treated separately. Personally,

My weekend is Saturday and Sunday. It is constant.

My first day of the week is Sunday. Even though my workday starts on Monday, I do my personal planning on Sunday because I have free time on that day. This "first day of the week" can vary because some people may choose Saturday or Monday.

**Comment by herrbrixter** [comment]
I like this idea. There should be some visual border between weeks.
